sync-diff-inspector/route-diff.md

```diff
@@ -52,12 +52,154 @@ To check a large number of tables with different schema names or table names, yo
     password = ""
 ########################### Routes ###########################
 [routes.rule1]
-schema-pattern = "test_1"      # Matches the schema name of the data source. Supports the wildcards "*" and "?"
-table-pattern = "*"            # Matches the table name of the data source. Supports the wildcards "*" and "?"
+schema-pattern = "test_1"      # Matches the schema name of the data source. Supports the wildcards "*" and "?".
+table-pattern = "*"            # Matches the table name of the data source. Supports the wildcards "*" and "?".
 target-schema = "test_2"       # The name of the schema in the target database
 target-table = "t_2"           # The name of the target table
 ```
 
-## Note
+## The initialization of table routers and some examples
+
+Suppose that you have a rule pattern in the configuration as follows:
+
+```toml
+[routes.rule1]
+schema-pattern = "schema*"  # the schema to match. Support the wildcard characters "*" and "?".
+table-pattern = "table_*"   # the table to match. Support the wildcard characters "*" and "?".
+target-schema = "schema"    # the target schema
+target-table = "table"      # the target table
+```
+
+The preceding rule matches `schema2.table_3` to `schema.table`.
+
+### The initialization of table routers
+
+If a `target-schema/target-table` table named `schema.table` exists in the rules,
+
+- If there is a rule that matches `schema.table` to `schema.table`, it is OK.
+
+-  If there is no rule that matches `schema.table` to `schema.table`, sync-diff-inspector will add a new rule `schema.table -> _no__exists__db_._no__exists__table_` to the table router. After that, sync-diff-inspector will treat the table `schema.table` as the table `_no__exists__db_._no__exists__table_`.
+
+If `target-schema` exists only in the rules as follows,
+
+```toml
+[routes.rule1]
+schema-pattern = "schema_2"  # the schema to match. Support wildcard characters * and ?.
+target-schema = "schema"     # the target schema
+```
+
+- If there is no table `schema` in the upstream, it is OK.
+
+- If there is a table `schema` in the upstream, and a rule matches the table, it is OK.
+
+- If there is a table `schema` in the upstream, but no rule matches the table, sync-diff-inspector will add a new rule `schema -> _no__exists__db_` to the table router. After that, sync-diff-inspector will treat the table `schema` as the table `_no__exists__db_`.
+
+If `target-schema.target-table` does not exist in the rules, sync-diff-inspector will add a rule to match`target-schema.target-table` to `target-schema.target-table` to make it case-insensitive, because the table router is case-insensitive.
```

Comment:
```suggestion
If `target-schema.target-table` does not exist in the rules, sync-diff-inspector will add a rule to match `target-schema.target-table` to `target-schema.target-table` to make it case-insensitive, because the table router is case-insensitive.
```